website/.gitignore

```diff
@@ -0,0 +1,23 @@
+# Dependencies
+/node_modules
```

Comment:
I think it's better to have directory specific ones. In case you move away from Docusaurus in future you don't have to touch your root `.gitignore`. Anyway another important directory to add is the `build` directory. Which approach would you prefer? I would recommend the website-specific `.gitignore` within the `website` directory.

Reply:
The rest of our repo mostly uses only the toplevel .gitignore file so I'd rather go for consistency here even though that's a good point.

Reply:
Ok have done that!